JWBestPracticeApps/AutoplayVideoFeed/AutoplayVideoFeedViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  AutoplayVideoFeedViewController.swift
+//  AutoplayVideoFeed
+//
+//  Created by Stephen Seibert  on 3/31/20.
+//  Copyright © 2020 Karim Mourra. All rights reserved.
+//
+
+import UIKit
+
+class AutoplayVideoFeedViewController: UITableViewController {
```

Comment:
Would it be possible for `AutoplayVideoFeedViewController` to inherit from the regular `FeedViewController` and only add the additional adjustments? Is this something that would make sense or are the 2 apps too different ?

Reply:
Interesting thought. My only question would be if that would obfuscate the code for those looking at it? Written in this fashion:
- We can make adjustments to `FeedViewController` in the future, without affecting `AutoplayVideoFeedViewController`.
- All of the code related to the target is within this target, so 100% of the solution is visible within the folder for `AutoplayVideoFeed`.

Reply:
> We can make adjustments to FeedViewController in the future, without affecting AutoplayVideoFeedViewController.

I think we would want adjustments to  `FeedViewController` to affect `AutoplayVideoFeedViewController` since `FeedViewController` is the best practice for adding a player in a feed. If those best practices change, they also have to change in `AutoplayVideoFeedViewController`.
Say we deprecate an API and add a new one, we wouldn't to have to update all our targets; we would surely miss one.

> All of the code related to the target is within this target, so 100% of the solution is visible within the folder for AutoplayVideoFeed.

The pattern in the other targets is to inherit from a simpler app, that way you only display the code needed to add the specific functionaility. For example in the non-feed targets, BaseViewController is the only target that sets up a player. The other targets inherit from it and only add the code needed to introduce the functionality being demoed. This way the developer can get what he wants immediately. If he needs additional context he can refer to the superclass

Reply:
Sounds good. I'll make the change shortly.